challenges/day9_script.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
+
+const fileName = 'day9_input.txt';
+
+const weakNumber = 15690279;
```

Comment:
Probably could have returned this from the part 1 program, and used that return value for day 2 instead of hard-coding